doc/userguide/rules/websocket-keywords.rst

```diff
@@ -0,0 +1,26 @@
+WebSocket Keywords
+==================
+
+websocket.mask
+--------------
+
+A boolean to tell if the payload is masked.
```

Comment:
Ah I see this is a bool. But why not the mask value?

Reply:
This does not seem meaningful to me.
What seems meaningful is if the payload is masked or not.

Reply:
In general, keys are often used to identify malware / malware families, so I think we should make it available.

Reply:
ok